src/screens/scanning/ConfirmValueScreen.tsx

```diff
@@ -97,53 +93,44 @@ export default function ConfirmValueScreen({
-          <CurrencyInput // TODO: refactor currency input & validation with custom text input base components
-            value={voucherAmount}
-            onChangeValue={onChangeVoucherAmount}
-            renderTextInput={props => (
-              <TextInput
-                {...props}
-                onBlur={() => setIsActive(false)}
-                onFocus={() => setIsActive(true)}
-                style={isActive ? Styles.FormFieldFocus : Styles.FormField}
-                placeholderTextColor={Colors.midGray}
-                secureTextEntry={false}
-                autoCorrect={false}
-                autoCapitalize="none"
-                keyboardType="number-pad"
-                returnKeyType="done"
-                selection={{
-                  start: String(props.value).length,
-                  end: String(props.value).length,
-                }}
-                caretHidden
-              />
-            )}
-            prefix="$"
-            minValue={0}
-            separator="."
-            delimiter=","
-            precision={2}
-          />
-          <ErrorContainer>
-            {showZeroError ? (
-              <RedText>
-                <Body2Subtext>
-                  Voucher must be redeemed for more than $0.
-                </Body2Subtext>
-              </RedText>
-            ) : null}
-            {showExceedError ? (
-              <RedText>
-                <Body2Subtext>
-                  Voucher value exceeds maximum redemption limit.
-                </Body2Subtext>
-              </RedText>
-            ) : null}
-          </ErrorContainer>
-        </FormContainer>
+        <Body1TextSemibold>Voucher Value</Body1TextSemibold>
+        <CurrencyInput // TODO: refactor currency input & validation with custom text input base components
+          value={voucherAmount}
+          onChangeValue={onChangeVoucherAmount}
+          renderTextInput={props => (
+            <TextInput
+              {...props}
+              onBlur={onBlur}
+              onFocus={onFocus}
+              style={isActive ? fieldFocused : styles.fieldDefault}
+              placeholderTextColor={Colors.midGray}
+              secureTextEntry={false}
+              autoCorrect={false}
+              autoCapitalize="none"
+              keyboardType="number-pad"
+              returnKeyType="done"
+              selection={{
+                start: String(props.value).length,
+                end: String(props.value).length,
+              }}
+              caretHidden
+            />
+          )}
+          prefix="$"
+          minValue={0}
+          separator="."
+          delimiter=","
+          precision={2}
+        />
+        <ErrorContainer>
+          <Body2Subtext>
+            <RedText>
+              {showZeroError && 'Voucher must be redeemed for more than $0.'}
+              {showExceedError &&
+                'Voucher value exceeds maximum redemption limit.'}
```

Comment:
Possibly standardize process for displaying error messages. This screen has separate state variables for each error type, while others have one 'showInvalidError,' where the error message to be displayed is another state variable. Not sure which one is better, but it'd be good to be consistent.